docker-compose.yml

```diff
@@ -1,5 +1,17 @@
 version: "2.4"
 services:
+  database:
+    image: postgres:12.2
+    environment:
+      - POSTGRES_DB=zombiesurvival
```

Comment:
Any reason to not specify these through `.env`?

Reply:
In development, these defaults don't need to be secret, and I believe it presents a clear association between the `database` service and others that depend on it.

In production, values passed into the `.env` file will override these environment variables, so we can specify a unique password. However, that's only an additional layer of security, since the database should be behind a firewall and inaccessible to the outside world.